controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -1195,6 +1199,156 @@ def test_controller_get_capabilities(self):
         self.servicer.ControllerGetCapabilities(request, context)
 
 
+class TestControllerServerExpandVolume(AbstractControllerTest):
+
+    def get_create_object_method(self):
+        return self.servicer.ControllerExpandVolume
+
+    def get_create_object_response_method(self):
+        return csi_pb2.ControllerExpandVolumeResponse
+
+    def setUp(self):
+        super().setUp()
+        self.fqdn = "fqdn"
+        self.mediator = XIVArrayMediator("user", "password", self.fqdn)
+        self.mediator.client = Mock()
+
+        self.storage_agent = MagicMock()
+        self.storage_agent.get_mediator.return_value.__enter__.return_value = self.mediator
+
+        self.servicer = ControllerServicer(self.fqdn)
+
+        self.request = Mock()

```

Comment:
can't these (+ secrets) move higher up the `setUp` hierarchy?

Reply:
yep
done

Reply:
I only see you moved two lines. what about most of the other lines in this function?

Reply:
there is a massive change for it in [CSI-1601](https://github.com/IBM/ibm-block-csi-driver/blob/e3755778032f718e079b0392554acdb19836b9ba/controller/tests/controller_server/csi_controller_server_test.py#L19)

Reply:
ok